src/validation.cpp

```diff
@@ -2205,6 +2210,33 @@ static void AppendWarning(bilingual_str& res, const bilingual_str& warn)
 
 void CChainState::UpdateTip(const CBlockIndex* pindexNew)
 {
+    auto& func_name = __func__;
+    auto log_progress = [this, pindexNew](
+            const std::string& prefix,
+            const std::string& warning_messages) EXCLUSIVE_LOCKS_REQUIRED(::cs_main)
+    {
+        LogPrintf("%s%s: new best=%s height=%d version=0x%08x log2_work=%f tx=%lu date='%s' progress=%f cache=%.1fMiB(%utxo)%s\n",
+            prefix, func_name,
+            pindexNew->GetBlockHash().ToString(), pindexNew->nHeight, pindexNew->nVersion,
+            log(pindexNew->nChainWork.getdouble())/log(2.0), (unsigned long)pindexNew->nChainTx,
+            FormatISO8601DateTime(pindexNew->GetBlockTime()),
+            GuessVerificationProgress(m_params.TxData(), pindexNew),
+            this->CoinsTip().DynamicMemoryUsage() * (1.0 / (1<<20)),
+            this->CoinsTip().GetCacheSize(),
+            !warning_messages.empty() ? strprintf(" warning='%s'", warning_messages) : "");
+    };
+
+    // The contents of this function are either not relevant if we're not
```

Comment:
94e67d6 This comment seems unclear to me; remove "either" and s/otherwise/in that case/? (not sure)
```suggestion
    // The contents of this function are not relevant if we're not
```

Reply:
Done, thanks.